### DISCUSSION TITLE
Post/pre hooks for remove operations

### DISCUSSION THREAD

**Discussion by jakobwarnhjelm** [typegoose/typegoose#604] [Q&A] [answered]
I'm able to get pre/post hooks working for save operations, as described here https://typegoose.github.io/typegoose/docs/guides/quick-start-guide/#hooks 

However I can't manage to create a hook for when `remove` or similar is used, see e.g. https://mongoosejs.com/docs/middleware.html#naming 

I would be very thankful for an example, including how to tell mongoose to distinguish between document and query hooks.

**Comment by hasezoey** [comment, marked as answer]
not possible currently, see #605 

or do you mean something else?

**Reply by jakobwarnhjelm** [reply]
That was quick, #605 would most likely solve my problem
